charts/bindplane/templates/bindplane.yaml

```diff
@@ -392,17 +383,8 @@ spec:
             capabilities:
               drop: ["ALL"]
           volumeMounts:
-            - name: {{ include "bindplane.fullname" . }}-prometheus-config
-              mountPath: /etc/prometheus/prometheus.yml
-              subPath: prometheus.yml
-            - name: {{ include "bindplane.fullname" . }}-prometheus-config
-              mountPath: /etc/prometheus/rules.yml
-              subPath: rules.yml
-            - name: {{ include "bindplane.fullname" . }}-prometheus-config
-              mountPath: /etc/prometheus/web.yml
-              subPath: web.yml
             - name: {{ include "bindplane.fullname" . }}-prometheus-data
-              mountPath: /var/lib/prometheus/tsdb
+              mountPath: /prometheus
```

Comment:
/prometheus is the correct path for the bindplane-prometheus image. See the bottom of this PR https://github.com/observIQ/bindplane-op-enterprise/pull/2084/files